platform/viewer/src/App.js

```diff
@@ -79,23 +87,20 @@ window.store = store;
 
 class App extends Component {
   static propTypes = {
-    routerBasename: PropTypes.string.isRequired,
-    servers: PropTypes.object.isRequired,
-    //
     oidc: PropTypes.array,
     whiteLabelling: PropTypes.object,
-    extensions: PropTypes.arrayOf(
-      PropTypes.shape({
-        id: PropTypes.string.isRequired,
-      })
-    ),
-    hotkeys: PropTypes.array,
+    routerBasename: PropTypes.string.isRequired,
+    config: PropTypes.func.isRequired,
+    defaultExtensions: PropTypes.array,
   };
 
   static defaultProps = {
     whiteLabelling: {},
     oidc: [],
-    extensions: [],
+    config: {
+      extensions: [],
+    },
```

Comment:
`whiteLabelling`, `oidc`, etc. are config properties. Why the addition of a wrapper object? Everything this component receives is technically application configuration.

Reply:
Updated, now I think its clearer.